lib/std/fs/wasi.zig

```diff
@@ -62,6 +78,15 @@ pub const Preopen = struct {
     }
 };
 
+/// WASI resource identifier struct. This is effectively a path within
+/// a WASI Preopen.
+pub const PreopenURI = struct {
```

Comment:
This is a stylistic nit more than anything else:

```suggestion
pub const PreopenUri = struct {
```